circleguard/gui.py

```diff
@@ -1096,10 +1400,26 @@ def switch_theme(dark, accent=QColor(71, 174, 247)):
         dark_p.setColor(QPalette.LinkVisited, accent)
 
         app.setPalette(dark_p)
-        app.setStyleSheet("QToolTip { color: #ffffff; "
-                          "background-color: #2a2a2a; "
-                          "border: 1px solid white; }"
-                          "QLabel {font-weight: Normal; }")
+        app.setStyleSheet("QToolTip {\n"
+                            "\tcolor: #ffffff;\n"
+                            "\tbackground-color: #2a2a2a;\n"
+                            "\tborder: 1px solid white;\n"
+                          "}\n"
+                          "QLabel {\n"
+                            "\tfont-weight: Normal;\n"
+                          "}\n"
+                          "QTextEdit {\n"
+                            "\tbackground-color: #212121;\n" # my vscode terminal color
+                          "}\n"
+                          "LoadableW {\n"
+                            "\tborder: 1.5px solid #272727\n"
+                          "}\n"
+                          "CheckW {\n"
+                            "\tborder: 1.5px solid #272727\n"
+                          "}\n"
+                          "DragWidget {\n"
+                            "\tborder: 1.5px solid #272727\n"
+                          "}\n")
```

Comment:
fix indentation (also below)

Reply:
what's the issue here? I formatted it to be how it would appear in a .css file. More readable imo.

Reply:
other than the linter screaming at my face I would prefer using a multiline string.
![](https://i.fiery.me/wNsYI.png)
those `\n` are horrible

Reply:
ah that's much better than using `\n` and `\t`, kind of forgot about multilines